canonical/maas-ui
# chore: Refactor SpaceDetails to use PageContent MAASENG-1825

## Done

- Refactored SpaceDetails to use PageContent

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to subnets and click "space", then click on a space
- Click "Delete space"
- Ensure the side panel opens correctly

## Fixes

Fixes [MAASENG-1825](https://warthogs.atlassian.net/browse/MAASENG-1825)


[MAASENG-1825]: https://warthogs.atlassian.net/browse/MAASENG-1825?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
